api-service/crowdsource-webapp/src/app/answers-distribution-chart/answers-distribution-chart.component.ts

```diff
@@ -0,0 +1,123 @@
+import { AfterViewInit, Component, ElementRef, Input, ViewChild, ViewEncapsulation } from '@angular/core';
+import { ActivatedRoute, Params, ParamMap, Router } from '@angular/router';
+import {
+  AnswerSummary,
+  CrowdsourceApiService,
```

Comment:
I don't see where "CrowdsourceApiService" is meaningfully used. It's given as to constructor, but never used as far as I can see...

Reply:
Ah, this was leftover from when I copied another component. removing.